### PR TITLE
Fix failing UI tests on CI

### PR DESCRIPTION
**Pull request by odockal**
Signed-off-by: Ondrej Dockal <odockal@redhat.com>

### CODE REVIEW

File: test/ui-test/initializationUITest.ts [hunk 1]
```diff
@@ -9,12 +11,16 @@ import { getNotifications, cleanUpNotifications } from './common/testUtils';
...
-  before(async () => {
+  before(() => {
     driver = VSBrowser.instance.driver;
-    await cleanUpNotifications();
   });
 
   describe('Knative view', () => {
+    before(async function setup() {
+      this.timeout(10000);
+      await cleanUpNotifications();
```
Comment (joshuawilson) [inline test/ui-test/initializationUITest.ts:21]: Why did you need to separate this? Both `before` functions get called before the test. Now you just have to repeat it every time. OR do you ONLY need it before this test?

Comment (odockal) [reply test/ui-test/initializationUITest.ts:21]: I was desperate, test was failing inside NotificationCenter in the lib. and I could not reproduce, very unstable. OS moving out and there. I will clear that up.

**Review by joshuawilson — APPROVED** [review]
